docs/components/eyebrow/test.html

```diff
@@ -0,0 +1,271 @@
+---
+title: Testing - Eyebrow
+crumbs:
+  - path: components/eyebrow
+    label: Eyebrow
+---
+{% extends 'test.njk' %}
+{% set contentClasses = 'Eyebrow-spec' %}
+
+{% macro dropdown(id='', isDirty=false, className='', isRequired=false, isDisabled=false, isFocused=false, isValid=true) %}
```

Comment:
Please remove the test page content.  

See: https://deploy-preview-707--helix-ui.netlify.app/components/switch/test

Reply:
Changes have been done, Please review